repos/system_upgrade/el7toel8/actors/networkmanagerreadconfig/actor.py

```diff
@@ -0,0 +1,42 @@
+from leapp.actors import Actor
+from leapp.models import NetworkManagerConfig
+from leapp.tags import IPUWorkflowTag, FactsPhaseTag
+import subprocess
```

Comment:
please move python standard library imports on top and sorted please.

See: https://www.python.org/dev/peps/pep-0008/#imports

> Imports should be grouped in the following order:
> 
> * Standard library imports.
> * Related third party imports.
> * Local application/library specific imports.
>
> You should put a blank line between each group of imports.